classes/Provider/Gateways/WebGatewayProvider.php

```diff
@@ -49,6 +51,20 @@
 {
     public function register(Container $app)
     {
+        $app['mailer'] = function ($app) {
```

Comment:
I'm not sure if the `WebGatewayProvider` is the right spot for this service. It's not really web functionality.

Reply:
@derrabus I am open to a suggestion on a better place to do this -- I have no strong opinion and I do not believe we established a place for non-web-specific services to be registered.

Should this be a service provider that gets registered in our Application object? (say, call it SwiftmailServiceProvider)?

Reply:
Since we're replacing the original `SwiftmailerServiceProvider`, I'd go for naming our provider `SwiftmailerServiceProvider` as well.

Alternatively, we could rename `ResetEmailerServiceProvider` to `MailerServiceProvider` and move the Swiftmailer definition there, so everything related to sending emails is in one provider.